client/app/lib/views/credentiallist/accountcredentiallist.coffee

```diff
@@ -24,14 +24,22 @@ module.exports = class AccountCredentialList extends KodingListView
   showCredential: (options = {}) ->
 
     { credential, cred } = options
+    
+    content = "<p><pre><code>#{cred}</code></pre></p>"
+    
+    scrollView = new kd.CustomScrollView { cssClass : 'readme-scroll' }
+    
+    scrollView.wrapper.addSubView markdown_content = new kd.CustomHTMLView
```

Comment:
the variable `markdown_content` doesn't seem to be used in this context, pls get rid of it.